lualib/resty/events/worker.lua

```diff
@@ -104,6 +125,98 @@ function _M.new(opts)
     return setmetatable(self, _MT)
 end
 
+local function read_thread(self, broker_connection)
+    while not terminating(self) do
+        local data, err = broker_connection:recv_frame()
+        if err then
+            if not is_timeout(err) then
+                return err_if_not_exiting("failed to read event", err)
+            end
+
+            -- timeout
+            goto continue
```

Comment:
Just wonder if we can add a log saying the worker is timeout connecting to the broker.

Reply:
No, we don't want to flood logs with something that happens all the time. E.g. when there is no events to be read, the recv_frame is just timeouting all the time every 1 seconds.